README.md

```diff
@@ -1,6 +1,63 @@
+
+Browsing with the extension loaded will behave normally, until the circle that will show up on the right side of the URL field is clicked. Clicking it once will put the ad-blocker to work doing it's best to remove advertisements and give you a preview of what content will be censored by coloring it red. Clicking it a second time engage censoring and replace the red text with content the censorer (server) deems as friendly but still contextually relevant.
+
+### Server
+
+#### Quick start with example
+
+Run `source server/activate`, then `server/scrape-postive -t && server/scrape-negative && server/run-backend` and use the client as described above when scripts are ready.
+
+#### Detailed guide
+
+Before using any of the server's functionality, `source` the `activate` file in `server/` in your `bash` or `zsh` shell. This will load the virtual environment the server lives in and greet you with the `(weBlock-server)` message in your shell's prompt.
+
+weBlock's server side is managed by three executable scripts in the `server/` directory, namely `scrape-positive`, `scrape-negative` and `run-backend`.
+
+##### Data collection & building models: scraping & training
+
+For the collection of data, weBlock relies on [Google News](https://news.google.com/) to scrape recently published articles and those articles to scrape information used to train its natural language processing models.
+
+`scrape-negative` is used to collect examples for what is undesired by the censorer. It searches Google News with the comma-separated queries defined in the environment variable `NEGATIVE_QUERIES` in `server/.env`.\
+The scraped articles are then used in censoring as negative examples, where the [Word Mover's Distance](http://proceedings.mlr.press/v37/kusnerb15.pdf) of a paragraph to the scraped article's summaries plays a role in determining wether that paragraph should be censored.
+
+`scrape-positive` is used to collect examples for what is desired by the censorer. It, analogously to `scrape-negative`, searches Google News with the comma-separated queries defined in the environment variable `POSITIVE_QUERIES` in `server/.env`.\
+If `scrape-positive` is run with the argument `-t` or `--train`, the resulting articles from this scraping are used to train a [Biterm Topic Model](https://github.com/xiaohuiyan/xiaohuiyan.github.io/blob/master/paper/BTM-WWW13.pdf) with the parameters defined by the environment variables `TRAINING_*` as given in `server/.env`. Training is necessary on the first run, but can later be skipped to reuse the existing BT Model.\
+The Biterm Topic Model makes the key decision in finding which of the scraped positive, desired examples in the database will be used to replace a paragraph that is marked for censorship.
+
+Both `scrape-positive` as well as `scrape-negative` have an optional argument `-n` or `--narticles` that can be used to define an upper limit for how many arguments are scraped per query.
```

Comment:
state the default values here too

Reply:
I think this would assume too much knowledge about BTM. IMO it makes more sense to expect someone that understands BTM / has read the linked paper to check the .env.example file than to add an overly long paragraph stating all of our default parameters that someone who hasn't researched BTM won't understand anyways.

Reply:
I'll add it for `-n` though, not sure if you meant that and / or the BTM training parameters🙈

Reply:
I'm also mentioning the default BTM parameters now as being 'fast but far-from-optimal'. Maybe something along the lines of that is what you meant